gamestonk_terminal/stocks/stocks_controller.py

```diff
@@ -189,6 +192,43 @@ def call_reset(self, _):
         return MENU_RESET
 
     # COMMANDS
+    @try_except
+    def call_search(self, other_args: List[str]):
+        """Process search command"""
+        parser = argparse.ArgumentParser(
+            add_help=False,
+            formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+            prog="search",
+            description="Show companies matching the search query.",
+        )
+
+        parser.add_argument(
```

Comment:
Also styling point moving forward (can blame @DidierRLopes ) : no newlines between parser arguments.

Reply:
And Done!